### PR TITLE
Use class for storing commands descriptions

### PR DESCRIPTION
**Pull request by ksdfg**
Class objects can now be reused for both command previews as well as help texts, centralising the process.

### CODE REVIEW

**Comment by ksdfg** [comment]
@ksdfg #all

File: telebot/utils.py [hunk 1]
```diff
@@ -15,6 +15,41 @@
 from telebot.modules.db.users import add_user, get_user
 
 
+# Class to represent all the info required to describe how to use a command
+class CommandDescription:
+    command: str
```
Comment (akhilnarang) [inline telebot/utils.py:20]: You can drop these 5 declarations

**Review by akhilnarang — CHANGES_REQUESTED** [review]

File: telebot/utils.py (hunk 1 above)
Comment (ksdfg) [reply telebot/utils.py:20]: ah gg, golang habits

Comment (akhilnarang) [reply telebot/utils.py]: LMAO

**Pull request merged**